doc/radosgw/metrics.rst

```diff
@@ -75,10 +75,17 @@ The following metrics related to S3 or Swift operations are tracked per Ceph Obj
      - Guage
      - Total latency of list bucket operations
 
-More information about op metrics can be seen in the ``rgw_op`` section of the output of the ``counter schema`` command.
-To view op metrics in the Ceph Object Gateway go to the ``rgw_op`` section of the output of the ``counter dump`` command::
+There are three different sections in the output of the ``counter dump`` and ``counter schema`` commands that show the op metrics and their information.
+The sections are ``rgw_op_global``, ``rgw_op_user``, and ``rgw_op_bucket``.
 
-    "rgw_op": [
+The counters in the ``rgw_op_global`` section reflect the totals of each op metric for a given Ceph Object Gateway.
+The counters in the ``rgw_op_user`` and ``rgw_op_bucket`` sections are labeled counters of op metrics for a user or bucket respectively.
```

Comment:
```suggestion
The counters in the ``rgw_op_user`` and ``rgw_op_bucket`` sections are labelled counters of op metrics for a user or bucket respectively.
```

Reply:
Looks like both spellings are correct, it's just a matter of British vs American English: https://www.grammarly.com/blog/labeled-labelled/

Reply:
@zdover23 do you have an opinion on this spelling?

Reply:
exactly, we should try to lean into on spelling dialect

Reply:
FWIW `labelled` looks wrong to me, and we've tended toward Murican English in the past for docs, so I vote to leave it as it is.

Reply:
Even though I live in the Commonwealth, I think that American English is in ascendancy. "labelled".